libs/ui/elements/src/lib/metadata-info/linkify.directive.ts

```diff
@@ -75,6 +75,6 @@ export class GnUiLinkifyDirective implements OnInit {
   }
 
   private createLink(displayValue: string, url: string): string {
-    return `<a href="${url}" target="_blank" class="text-primary cursor-pointer hover:underline">${displayValue} <mat-icon class="material-symbols-outlined !w-[12px] !h-[14px] !text-[14px] opacity-75">open_in_new</mat-icon></a>`
+    return `<a href="${url}" target="_blank" class="text-primary cursor-pointer hover:underline"><mat-icon class="material-symbols-outlined mr-1 !w-[12px] !h-[14px] !text-[14px] opacity-75">open_in_new</mat-icon> ${displayValue}</a>`
```

Comment:
Just to make sure, is this change desired everywhere the directive is used in the new design?

Reply:
The directive isn't visibly used anywhere else in the mockups, so it's just me assuming here.